ses

```diff
@@ -339,7 +327,14 @@ fi
 #############################################################
 section_header "Ceph related services"
 
-plugin_command "systemctl status -l 'ceph*'"
+if [ -x /usr/sbin/cephadm ]; then
+    cephadm ls | jq -r '.[] | "\(.fsid) \(.name)"' |
```

Comment:
looks odd to first query for `/usr/sbin/cephadm` but then use `cepham` to execute the commands. Maybe make them symmetric? Either call `$(which cephadm)` or run `/usr/sbin/cephadm ls`?

Reply:
Fixed in 285747b